myhpi/static/css/myHPI.css

```diff
@@ -205,19 +205,11 @@ h1.toc-title::after {
     height: 0px;
 }
 
-.print-shown {
+.d-print-block {
     display: none;
 }
 
 @media print {
-    .print-none {
-        display: none !important;
-    }
-
-    .print-shown {
-        display: block !important;
-    }
-
     nav, .toc, .toc-container {
         display: none;
     }

```

Comment:
We could use `d-print-none` here

Reply:
Edit: At least for nav you are already using it

Reply:
Problem with toc is that it is generated, there is no .toc element in the template that I could assign the class too. Of course you could add the class in the markdown parser that generates it, but at that point I like the css solution better.

Reply:
Ahh, yeah, agreed!